utils/src/main/scala/it/pagopa/interop/commons/utils/service/OffsetDateTimeSupplier.scala

```diff
@@ -5,5 +5,5 @@ import java.time.OffsetDateTime
 /** Supplies a timestamp through DI
   */
 trait OffsetDateTimeSupplier {
-  def get: OffsetDateTime
+  def get(): OffsetDateTime
```

Comment:
These parentheses were a desired thing for me since a while ago. Adding them enables us to implement this trait like 
```scala
val foo: OffsetDateTimeSupplier = () => OffsetDateTime.now()
```
instead of having to create a new class from scratch like explicitly
```scala
val foo: OffsetDateTimeSupplier = new OffsetDateTimeSupplier {
   override def get: UUID = () => OffsetDateTime.now()
}
```
IT MAY BE NON-RETROCOMPATIBLE THOUGH. The fix, in that case, is adding a couple of parentheses to the call.